modules/nested-tabs-html/assets/scss/frontend.scss

```diff
@@ -1,6 +1,48 @@
+@import "../../../../assets/dev/scss/helpers/mixins";
+
 //
 // Nested Tabs
 //
+
+@mixin active-tab-title() {
+
+	&,
+	& a {
+		color: var( --n-tabs-title-color-active );
```

Comment:
This was how we did it originally. You can check in the current nested tabs widget. We use spaces everywhere. I didn't want to change it, so that we don't see a lot of unrelated changes.